data/tsv/cat_latn_phonemic.tsv

```diff
@@ -3965,55 +3965,6 @@ alvocats	ə l b u k a t s
 alvocats	ə l v o k a t s
 alzina	a l z i n a
 alzina	ə l z i n ə
-al·legada	a l l e ɡ a d a
```

Comment:
These words should not be excluded:

https://en.wikipedia.org/wiki/Catalan_orthography#Punt_volat_(middot)